packages/common/core-interfaces/src/test/brandedType.spec.ts

```diff
@@ -3,6 +3,8 @@
  * Licensed under the MIT License.
  */
 
+import type { BrandedType } from "@fluidframework/core-interfaces/internal";
```

Comment:
Also like in the other PR, the eslint disable comment didn't come along with the move.

Reply:
With the updated import plugin, looks like the comment isn't needed. Good catch.